File: BOJ/Bronze/2525.js
// 오븐시계

const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : '../input.txt';
const input = fs.readFileSync(filePath).toString().split('\n');
let [hour, minute] = input[0].split(' ');
let takeTime = input[1];

let answer = '';

let sum = Number(minute)+Number(takeTime);


if( sum > 59) {
    let t = parseInt(sum/60);
    hour = Number(hour)+t;
    sum -= t*60;
}
if(hour > 23) hour -= 24;
answer = hour + " " + sum;

console.log(answer);

File: BOJ/Bronze/2525.py
ch, cm = map(int,input().split())

cost = int(input())

nm = cm+cost
nh = ch
if nm > 59 :
  nh += (nm//60)
  nm = (nm%60)

  if nh > 23 :
    nh -= 24

print(f'{nh} {nm}')